src/logos/mercadopago/logo.jsx

```diff
@@ -0,0 +1,128 @@
+/* @flow */
+/** @jsx node */
+
+import { node, type ComponentNode } from 'jsx-pragmatic/src';
+
+import { SVGLogo, getLogoColors, type SVGLogoProps } from '../../lib';
+import { LOGO_COLOR, LOGO } from '../../constants';
+import { type LogoColorMap } from '../../types';
+
+const LOGO_COLORS : LogoColorMap = {
+    [ LOGO_COLOR.DEFAULT ]: {
+        primary:   '#2D3277',
+        secondary: '#009EE3',
+        tertiary:  '#FFFFFF'
+    },
+    [ LOGO_COLOR.WHITE ]: {
+        primary:   '#FFFFFF',
+        secondary: '#FFFFFF',
+        tertiary:  '#FFFFFF'
+    },
+    [ LOGO_COLOR.BLACK ]: {
+        primary:   '#2D3277',
+        secondary: '#009EE3',
+        tertiary:  '#FFFFFF'
+    }
+};
+
+// eslint-disable-next-line flowtype/require-exact-type
+export function MercadoPagoLogo({ logoColor = LOGO_COLOR.DEFAULT, ...props } : { logoColor? : $Values<typeof LOGO_COLOR> }) : ComponentNode<SVGLogoProps> {
+
+    const { primary, secondary, tertiary } = getLogoColors(LOGO.MERCADOPAGO, LOGO_COLORS, logoColor);
+
+    return (
+        <SVGLogo
+
+            { ...props }
+            name={ LOGO.MERCADOPAGO }
+            logoColor={ logoColor }
+            render={ () => {
+                switch (logoColor) {
+                case LOGO_COLOR.WHITE:
```

Comment:
Please don't add a new SVG for every color -- this will result in a large SDK size. Please follow the example for the other logos in this repo.

Reply:
In this case, the SVGs that were provided to us are different for the white and default version. I saw that we had a similar case in https://github.com/paypal/paypal-sdk-logos/blob/master/src/logos/credit/logo.jsx. Is the recommendation here to go back to our designer and request the same SVG for different color cases?

Reply:
Oh, are they significantly different logos? I would recommend condensing into one logo if possible -- even in the credit case, it's not ideal, and I'm hoping to find a better way there.